pkgs/applications/networking/cluster/k9s/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  name = "kubernetes-k9s-${version}";
+  version = "0.7.13";
+
+  src = fetchFromGitHub {
+    owner = "derailed";
+    repo = "k9s";
+    rev = version;
+    sha256 = "0wsj6wc2qi5708cg47l2qblq1cg8fcwxdygpkayib9hapx6lc6f8";
+  };
+
+  modSha256 = "1ia9wx6yd9mdr981lcw58xv39iqzz25r03bmn1c6byxmq2xpcjq8";
+
+  subPackages = [ "." ];
```

Comment:
Is this necessary?

Reply:
Dont know tbh.

Reply:
Can be removed if you are sure this is not needed.

Reply:
Lets try build without this.